install/operator/pkg/cmd/internal/install/install.go

```diff
@@ -245,16 +250,23 @@ func (o *Install) render(fromFile string) ([]unstructured.Unstructured, error) {
 	if o.addons != "" {
 		addons = strings.Split(o.addons, ",")
 	}
-
+	productized, err := strconv.ParseBool(pkg.Productized)
+	if err != nil {
+		log.Info("productized", "Unable to determine PRODUCTIZED parameter, setting as false. Err: ", err)
+		productized = false
+	}
+	productVersion := configuration.ProductVersionOfTag(pkg.DefaultOperatorTag)
 	resources, err := generator.Render(fromFile, RenderScope{
-		Namespace:     o.Namespace,
-		Image:         o.image,
-		Tag:           o.tag,
-		DevSupport:    o.devSupport,
-		Role:          RoleName,
-		Kind:          "Role",
-		EnabledAddons: addons,
-		DatabaseImage: o.databaseImage,
+		Namespace:      o.Namespace,
+		Image:          o.image,
+		Tag:            o.tag,
+		DevSupport:     o.devSupport,
+		Role:           RoleName,
+		Kind:           "Role",
+		Productized:    productized,
+		ProductVersion: productVersion,
```

Comment:
So the Productized flag is being read in to RenderScope from a boolean set at compile-time not at runtime. This seems wrong since it is possible to read the Productized flag from config. Although we don't at the moment call [configuraiton.GetProperties](https://github.com/syndesisio/syndesis/blob/master/install/operator/pkg/syndesis/configuration/configuration.go#L308) in this install.Render function we do call it in the [before](https://github.com/syndesisio/syndesis/blob/master/install/operator/pkg/cmd/internal/install/install.go#L154) function. Therefore, it should be possible to extract the Productized flag from that configuration and add it to Renderscope (maybe using the o.Productized as the vehicle to hold the value migrating it to o.productized).